### PR TITLE
NODE-2635: Update Dockerfiles to be compatible with Bullseye

### PR DESCRIPTION
**Pull request by jcolekaplan**
This is an offshoot of https://github.com/Contrast-Security-OSS/NodeTestBenches/pull/310 which caused the pipeline to fail: https://github.com/Contrast-Security-OSS/NodeTestBenches/actions/runs/3061326288 because there is no Docker image of Node18 built for Debian "Stretch". I don't think there will be since it's no longer LTS: https://www.debian.org/releases/stretch/ but I couldn't find anything confirming or denying that. Regardless, I think it's best to get off it and use the current LTS which is "Bullseye": https://www.debian.org/releases/bullseye/

The main problem with that is mysql-client and mysql-server aren't on anything later than "Stretch": https://packages.debian.org/stretch/mysql-server (available) v. https://packages.debian.org/bullseye/mysql-server (not available) so we need to change it to mariadb-client and mariadb-server which also required updating some of the mysql commands.

Tl;dr: Upgrading one thing required upgrading another thing that required upgrading another thing

I've tested both updated Dockerfiles locally and they build and run. I uploaded the v5 one to ECR and ran screener tests on it and they all pass.

### CODE REVIEW

**Review by IvayloGrahlyov — APPROVED** [review]
Tested locally and it works ok :)
